src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+
+h2,
+p,
+ol {
+  margin-bottom: 15px;
+}
+
+a {
+  color: green;
+  font-size: 15px;
+}
+
+#other {
+  font-size: 15px;
+  font-weight: normal;
```

Comment:
no need to duplicate this rule.
`a` inherit `font-size` and `font-weight` from `h2`